server/public/layout.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable font-family-name-quotes */
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap");
```

Comment:
I didn't notice this before but `@import` is generally not considered good because it block parallel loading of stylesheets. Link it in your `<head>` instead